astrospice/kernel.py

```diff
@@ -102,3 +102,97 @@ def coverage(self, body):
         body = Body(body)
         coverage = [t for t in spiceypy.spkcov(self._fname_str, body.id)]
         return Time(coverage, format='et').utc
+
+class MetaKernel(KernelBase):
+    """
+    A class for a single .tm kernel.
+
+    References
+    ----------
+    https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/req/kernel.html#Additional%20Meta-kernel%20Specifications
+    """
+    _file_extension = '.tm'
+    
+    def __init__(self, fname):
+        self._fname = fname
+        self.change_path_value()
+        if self.kernels_exist:
+            self.load_kernels()
+        else:
+            print("Kernels are not yet loaded")
+        
+    def change_path_value(self):
+        """
+        Changes the PATH_VALUES parameter in the .tm file
+        to point to the same folder as the .tm file.
+        """
+        #open the file
+        with open(self.fname, 'r') as file:
+            # read a list of lines into data
+            data = file.readlines()
+
+        #find the line with PATH_VALUES = ( '..' )
+        for i, line in enumerate(data):
+            line_split = line.split()
+            if len(line_split) > 1 and line_split[0] == 'PATH_VALUES':
+                if line_split[-2] == "'..'":
+                    #replace the ".." with the folder path
+                    data[i] = data[i].replace('..', str(self.fname.parent))
+        
+        #write over the original file
+        with open(self.fname, 'w') as file:
+            file.writelines( data )
+            
+    @property
+    def kernels(self):
+        "List of kernels specified by the metakernel"
+        kernels = []
+        with open(self.fname, "r") as file:
+            look_for_kernels = False
+            for  line in file:
+                #split by whitespace
+                line_split = line.split()
+                
+                # now find the ) bracket by itself, this is the end of kernels to load
+                if len(line_split) >= 1 and line_split[0] == ')':
+                    look_for_kernels = False
+                    break
+                
+                if look_for_kernels == True and len(line_split) > 0:
+                    # find the filename for the kernel. 
+                    # The slicing removes the $KERNEL and the final '
+                    kernel_fname = line_split[0][9:-1]
+                    #do not add empty lines
+                    if kernel_fname != '':
+                        #slice removes the first \
+                        kernels.append(kernel_fname[1:])
+                
+                if len(line_split) > 1 and line_split[0] == 'KERNELS_TO_LOAD':
+                    look_for_kernels = True
+        
+        return kernels
+    
+    def load_kernels(self):
+        """
+        Loads the kernels specified by the metakernel
+        """
+        for kernel in self.kernels:
+            try:
+                Kernel(self.fname.parent / Path(kernel))
```

Comment:
```suggestion
                Kernel(self.fname.parent / kernel)
```
Given self.fname.parent is already a `Path`, anything else added with a `/` can be a string.